File: 2015/02/1.py
input = None
try:
    input = open("./2015/02/input.txt", "r")
except:
    print("Error opening input.txt")
    exit(1)

area = 0
for line in input.read().split("\n"):
    if line == "":
        continue
    sides = line.split("x")
    for i in range(len(sides)):
        sides[i] = int(sides[i])
    sides.sort()
    area += 3 * sides[0] * sides[1] + 2 * sides[2] * (sides[0] + sides[1])

print(area)

File: 2015/02/1.js
"use strict";
const fs = require("node:fs");

const input = fs.readFileSync("./2015/02/input.txt", "utf8").toString();

let area = 0;
for (const line of input.split("\n")) {
  if (line === "") continue;
  let sides = line
    .split("x")
    .map((side) => Number(side))
    .sort((a, b) => a - b);
  area += 3 * sides[0] * sides[1] + 2 * sides[2] * (sides[0] + sides[1]);
}

console.log(area);
